.github/ISSUE_TEMPLATE/release.md

```diff
@@ -8,44 +8,26 @@ assignees: ''
 ---
 
 ### List of pending issues / PRs
-* [ ] description #xxx https://github.com/eclipse/che/issues/xxx
+* pending issues/PRs are referenced in the respective Slack thread.
```

Comment:
which slack thread?

Reply:
fair, I will keep the old `template` string for referencing PRs/issues